README.md

```diff
@@ -1,24 +1,48 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Ruby Linter Templte
+![](https://img.shields.io/badge/Morse-Decorder-brown)
+# Morse Decoder
+> This project uses Ruby to decode morse code.
+
+## Built With
+- Ruby
+- Git
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+Clone the project by using terminal:
+```
+git clone git@github.com:ahmedtaa/morse-decode.git
+ruby decoder.rb
+```
+### Prerequisites
+-Ruby
 
 ## Authors
 
-👤 **Author1**
+👤 **Ahmed Alrakhawy**
+
+- GitHub: [Ahmed Alrakhawy](https://github.com/ahmedtaa)
+- LinkedIn: [Ahmed Alrakhawy](https://www.linkedin.com/in/ahmedtaa/)
+
+👤 **Benjmain Semah**
 
-- GitHub: [@ahmedta](https://github.com/ahmedta)
-- Twitter: [@\_ahmedta](https://twitter.com/_ahmedta)
+- GitHub: [@BenjaminSemah](https://github.com/BenjaminSemah)
+- Twitter: [@BenjaminSemah](https://twitter.com/BenjaminSemah)
+- LinkedIn: [@BenjaminSemah](https://www.linkedin.com/in/benjaminsemah/)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/ahmedtaa/morse-decode/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
+## Acknowledgments
+
+- Microverse
+
```

Comment:
- [ ] the link to the MIT license is not working as intended, please have another look at the [Microverse template](https://github.com/microverseinc/readme-template) for an example of how it is expected to work. please adjust it accordingly.

Reply:
RESOLVED: Readme file has been updated to include correct link to MIT License.